### DISCUSSION TITLE
[FR] <title>Status Code 401

### DISCUSSION THREAD

**Discussion by shahdnizar** [cjbarrie/academictwitteR#336] [General]
### Describe the solution you'd like

I keep getting 401 error when trying to use the function get_all_tweets: "Error in make_query(url = endpoint_url, params = params, bearer_token = bearer_token, : something went wrong. Status code: 401". I have checked my credentials and they worked fine with the rtweet package. What might be wrong?. Here is my code:
tweets <-
  get_all_tweets(
    query = "#BLM",
    start_tweets = "2020-01-01T00:00:00Z",
    end_tweets = "2020-01-05T00:00:00Z",
    bearer_token = get_bearer(),
    data_path = "data/",
    n = 1000,
  )
<img width="566" alt="Capture" src="https://user-images.githubusercontent.com/91488582/173201300-99867558-0d4b-4d7d-b48a-a01ec05f21cd.PNG">

**Comment by chainsawriot** [comment]
@shahdnizar You can't use your API1.1 bearer token (rtweet) for this package.

https://cran.r-project.org/web/packages/academictwitteR/vignettes/academictwitteR-auth.html

**Reply by shahdnizar** [reply]
Hi @chainsawriot , I do have access to an academic research account/credentials.
